plugins/modules/mso_schema_template.py

```diff
@@ -135,6 +135,8 @@ def main():
 
     mso = MSOModule(module)
```

Comment:
Can you revert this module to the original?

Reply:
done